assets/app.xmcl.voxelum.metainfo.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<component type="desktop-application">
+  <!-- Use reverse-DNS convention for ID -->
+  <id>app.xmcl.voxelum</id>
+  <name>X-Minecraft Launcher</name>
+  <summary>A modern Minecraft launcher with modpack support</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <developer_name>Voxelum</developer_name>
+
+  <url type="homepage">https://xmcl.app/ </url>
+  <url
+    type="bugtracker"
+  >https://github.com/Voxelum/x-minecraft-launcher/issues </url>
+  <url type="help">https://github.com/Voxelum/x-minecraft-launcher/wiki </url>
+
+  <description>
+    <p>
+      X-Minecraft Launcher (XMCL) is a feature-rich Minecraft launcher that lets you manage resources like modpacks, resource packs, mods, and shader packs. It integrates with Minecraft Forge, Fabric, Quilt, CurseForge, and Modrinth.
+    </p>
+    <p>Key features:</p>
+    <ul>
+      <li>Multi-instance management</li>
+      <li>Mod installation from CurseForge and Modrinth</li>
+      <li>Support for Forge, Fabric, and Quilt mod loaders</li>
+      <li>Resource pack and shader pack management</li>
+      <li>Multi-account support (Microsoft/Mojang/3rd-party)</li>
+      <li>Peer-to-peer multiplayer hosting</li>
+    </ul>
+  </description>
+
+  <launchable type="desktop-id">app.xmcl.voxelum.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <caption>Main interface showing instance management</caption>
+      <image
+      >https://raw.githubusercontent.com/Voxelum/xmcl-page/refs/heads/master/src/assets/store-installed.png </image>
+    </screenshot>
+    <screenshot>
+      <caption>Modpack browser with CurseForge integration</caption>
+      <image
+      >https://raw.githubusercontent.com/Voxelum/xmcl-page/refs/heads/master/src/assets/store.png </image>
+    </screenshot>
```

Comment:
fix the line breaks the tags are broken, also screenshot links should be from a tag or a commit not master